sdk/all/src/main/java/io/opentelemetry/sdk/OpenTelemetrySdkBuilder.java

```diff
@@ -51,19 +49,31 @@ public OpenTelemetrySdkBuilder setPropagators(ContextPropagators propagators) {
 
   /**
    * Returns a new {@link OpenTelemetrySdk} built with the configuration of this {@link
-   * OpenTelemetrySdkBuilder}.
+   * OpenTelemetrySdkBuilder} and registers it as the global {@link
+   * io.opentelemetry.api.OpenTelemetry}.
+   *
+   * @see GlobalOpenTelemetry
+   */
+  public OpenTelemetrySdk buildAndRegisterGlobal() {
+    OpenTelemetrySdk sdk = buildWithoutRegisteringGlobal();
+    GlobalOpenTelemetry.set(sdk);
+    return sdk;
+  }
+
+  /**
+   * Returns a new {@link OpenTelemetrySdk} built with the configuration of this {@link
+   * OpenTelemetrySdkBuilder}. This SDK is not registered as the global {@link
+   * io.opentelemetry.api.OpenTelemetry}. It is recommended that you register one SDK using {@link
+   * OpenTelemetrySdkBuilder#buildAndRegisterGlobal()} for use by instrumentation that requires
+   * access to a global instance of {@link io.opentelemetry.api.OpenTelemetry}.
+   *
+   * @see GlobalOpenTelemetry
    */
-  public OpenTelemetrySdk build() {
+  public OpenTelemetrySdk buildWithoutRegisteringGlobal() {
```

Comment:
I still think that just calling this `build()` would be fine. If you're looking at your auto-complete options, it should be obvious that the one that has `AndRegisterGlobal()` on the end of it would do what it says, and, hence, this one wouldn't.

Reply:
Ok :)